ArduPlane/Parameters.cpp

```diff
@@ -1491,7 +1485,12 @@ void Plane::load_parameters(void)
 #if AP_FENCE_ENABLED
     AP_Param::convert_class(g.k_param_fence, &fence, fence.var_info, 0, true);
 #endif
-  
+
+    // PARAMETER_CONVERSION - Added: July-2025 for Plane-4.7
```

Comment:
```suggestion
    // PARAMETER_CONVERSION - Added: July-2025 for ArduPilot-4.7
```

(doesn't need to be done on this PR, it's clear enough, but we should use this naming in future, I reckon)

Reply:
Fixed